config/default/webhookcainjection_patch.yaml

```diff
@@ -0,0 +1,15 @@
+# This patch add annotation to admission webhook config and
+# the variables $(CERTIFICATE_NAMESPACE) and $(CERTIFICATE_NAME) will be substituted by kustomize.
+# apiVersion: admissionregistration.k8s.io/v1
+# kind: MutatingWebhookConfiguration
+# metadata:
+#   name: mutating-webhook-configuration
+#   annotations:
+#     cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
+# ---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validating-webhook-configuration
+  annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
```

Comment:
I haven't had a chance to test this, but speaking from (distant) experience in setting up similar for DWO -- have we verified this works on both Kubernetes and OpenShift? I recall some differences in how certs for webhooks are managed on OpenShift vs. Kubernetes. On Kubernetes we can use `cert-manager` but on OpenShift we need (IIRC) the [service-ca](https://github.com/openshift/service-ca-operator).

Reply:
@dkwon17 
We have different configuration for Kubernetes and OpenShift.
Please have a look at [1]. As far as I remember I was inspired from DWO configuration.

[1] https://github.com/eclipse-che/che-operator/tree/a4e19ead9d324dce09f5ca400a5cb7eff3abd253/config/openshift/patches